src/qt/bitmap.cpp

```diff
@@ -237,6 +237,9 @@ void wxBitmap::InitFromImage(const wxImage& image, int depth, double WXUNUSED(sc
                  ? QBitmap::fromImage(ConvertImage(image), Qt::ThresholdDither)
                  : QPixmap::fromImage(ConvertImage(image, &mask));
 
+    if (mask)
+        qtImage.setMask(*mask->GetHandle());
+
```

Comment:
Keeping the _mask_ seperated from the _qtImage_ **is intentional** because this is how the other ports works. IOW, `setMask()` merges the mask with the pixmap's alpha channel. And AFAIK none of the other ports do this by default.
If you really want the mask be merged just be explicit and call `wxBitmap::QtBlendMaskWithAlpha()`